metpetdb_api/settings/staging.py

```diff
@@ -46,13 +46,14 @@
     'rest_framework',
     'rest_framework.authtoken',
     'django_extensions',
-    'vendor.djoser.djoser',
```

Comment:
Why have we moved from using a fork of Djoser?  This breaks authentication in the front end. The current implementation of  interface_v2 relies on a custom /login endpoint to make use of user_ids. https://github.com/kristallizer/djoser/commit/0e3b5d62d88cec8c6d949785dce487753d8edf59

to allow

`session["id"] = auth_token["user_id"]` in the front end

If we'd like to use a newer version of Djoser, then I suggest updating the fork instead.